fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/lao/event/rollcall/RollCallFragmentTest.kt

```diff
@@ -488,21 +520,65 @@ class RollCallFragmentTest {
       )
   }
 
+  @Test
+  fun sortedAttendeeListShowsNoToast() {
+    rollCallRepo.updateRollCall(LAO_ID, ROLL_CALL_SORTED_ATTENDEES)
+    rollCallRepo.updateRollCall(LAO_ID, closeRollCall(ROLL_CALL_SORTED_ATTENDEES))
+    InstrumentationRegistry.getInstrumentation().waitForIdleSync()
+    openRollCallWithDescription(ROLL_CALL_SORTED_ATTENDEES)
+
+    InstrumentationRegistry.getInstrumentation().waitForIdleSync()
+
+    RollCallFragmentPageObject.rollCallListAttendees()
+      .check(
+        ViewAssertions.matches(
+          ViewMatchers.withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)
+        )
+      )
+
+    UITestUtils.assertToastDisplayedHasNotText(R.string.roll_call_attendees_list_not_sorted)
+  }
+
+  @Test
+  fun unSortedAttendeeListShowsToast() {
+    rollCallRepo.updateRollCall(LAO_ID, ROLL_CALL_UNSORTED_ATTENDEES)
+    rollCallRepo.updateRollCall(LAO_ID, closeRollCall(ROLL_CALL_UNSORTED_ATTENDEES))
+    InstrumentationRegistry.getInstrumentation().waitForIdleSync()
+    openRollCallWithDescription(ROLL_CALL_UNSORTED_ATTENDEES)
```

Comment:
I think that this call here may actually be useless, it should be sufficient to update the repository (as long as you modify the roll call that is currently set in the fragment)

Reply:
I am not sure, tests do not pass the rollCallListAttendees visibility part if I don't put this line